client/components/payment-method/style.scss

```diff
@@ -0,0 +1,66 @@
+.payment-method {
+	background-image: url('/calypso/images/upgrades/cc-visa-disabled.svg');
+	background-position: 18px center;
+	background-repeat: no-repeat;
+	background-size: 60px auto;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	padding: 15px 15px 15px 95px;
+
+	.payment-method__name {
+		color: darken( $gray, 10% );
+	}
+
+	.payment-method__number {
+		color: $gray-dark;
+		flex-basis: 100%;
+		font-weight: 600;
+		text-transform: uppercase;
+	}
+
+	.payment-method__expiration-date {
+		color: darken( $gray, 10% );
+		font-style: italic;
+		opacity: 0.7;
+		white-space: nowrap;
+	}
+}
+
+input[type=radio].payment-method__radio.form-radio {
+	margin-top: 25px;
+}
+
+.payment-method.visa {
+	background-image: url('/calypso/images/upgrades/cc-visa-disabled.svg');
```

Comment:
If you can't get Webpack to copy the assets to the bundle folder, you should consider inlining the SVG here, using a loader. SVGs are very small and can be embedded as plain text (no base64 overhead), I think that would give you less headaches :)